spec/models/category_spec.rb

```diff
@@ -5,4 +5,15 @@
     it { is_expected.to have_db_column :id }
     it { is_expected.to have_db_column :name }
   end
+
```

Comment:
If we decide we want to have the attribute `:status` for the category model, then we should probably be validating the existence of the column here. This shouldn't be a problem as the journalist has to select a category on creation of each article.